src/sentry/static/sentry/app/components/todos.jsx

```diff
@@ -0,0 +1,312 @@
+import moment from 'moment';
+import React from 'react';
+import {t} from '../locale';
+
+import ApiMixin from '../mixins/apiMixin';
+import OrganizationState from '../mixins/organizationState';
+
+const TodoItem = React.createClass({
+  propTypes: {
+    task: React.PropTypes.object,
+  },
+
+  mixins: [OrganizationState],
+
+  getInitialState: function() {
+    return {
+      showConfirmation: false
+    };
+  },
+
+  toggleConfirmation: function() {
+    this.setState({showConfirmation: !this.state.showConfirmation});
+  },
+
+  skip: function(task) {
+    this.props.onSkip(task);
+    this.setState({showConfirmation: false});
+  },
+
+  render: function() {
+    let org = this.getOrganization();
+
+    let classNames = '';
+    let description = '';
+    let doneDescription = 'By being here, you\'ve done it. Welcome to Sentry!';
+    if (this.props.task.status == 'Complete') {
+      classNames += ' checked';
+
+      if (this.props.task.task == 0) {
+        description = doneDescription;
+      } else {
+        description = this.props.task.user + ' completed this ' + moment(this.props.task.date_completed).fromNow();
+      }
+
+    } else if (this.props.task.status == 'Pending') {
+      classNames += ' pending';
+      description = this.props.task.user + t(' kicked this off ') + moment(this.props.task.date_completed).fromNow();
+    } else if (this.props.task.status == 'Skipped') {
+      classNames += ' skipped';
+      description = this.props.task.user + t(' skipped this ') + moment(this.props.task.date_completed).fromNow();
+    } else {
+      description = this.props.task.description;
+    }
+
+    if (this.state.showConfirmation) {
+      classNames += ' blur';
+    }
+
+    let learn_more_url = '';
+    if (this.props.task.feature_location === 'project') {
+      learn_more_url = '/organizations/' + org.slug + '/projects/choose/?next=' + this.props.task.location;
+    } else if (this.props.task.feature_location === 'organization') {
+      learn_more_url = '/organizations/' + org.slug + '/' + this.props.task.location;
+    } else if (this.props.task.feature_location === 'absolute') {
+      learn_more_url = this.props.task.location;
+    }
+
+    return (
+      <li className={classNames}>
+        { this.props.task.status == 'Pending' ? <span className="pending-bar" /> : null }
+        <div className="todo-content">
+          <div className="ob-checkbox">
+            { this.props.task.status == 'Complete' ? <span className="icon-checkmark" /> : null }
+            { this.props.task.status == 'Skipped' ? <span className="icon-x" /> : null }
+            { this.props.task.status == 'Pending' ? <span className="icon-ellipsis" /> : null }
```

Comment:
Note that you can do this in JSX:

```javascript
{ this.props.task.status == 'Complete' && <span className="icon-checkmark"/> }
{ this.props.task.status == 'Skipped' && <span className="icon-x" /> }
{ this.props.task.status == 'Pending' && <span className="icon-ellipsis" /> }
```

If that statement evaluates to `false`, JSX converts `{false}` to [nothing / no DOM node](https://facebook.github.io/react/tips/false-in-jsx.html).